NomadAudio/src/PathUtils.cpp

```
@@ -0,0 +1,142 @@
+    
+    std::string utf8(static_cast<size_t>(utf8Len), '\0');
+    WideCharToMultiByte(
+        CP_UTF8, 0,
+        wide.c_str(), static_cast<int>(wide.size()),
+        utf8.data(), utf8Len, nullptr, nullptr
+    );
+    return utf8;
+#else
+    // On Unix-like systems, wstring is typically UTF-32; do simple narrowing
+    // (Real code should use proper iconv or similar)
+    std::string result;
```

Clang (14.0.6)

[warning] variable 'result' is not initialized

(cppcoreguidelines-init-variables)